web/src/components/__tests__/__snapshots__/LocationFooter.spec.tsx.snap

```diff
@@ -1,7 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It would be really nice if you could replace this snapshot test with a unit test. We said we want to gradually move away from those and replace them if they need to be adjusted.

Reply:
well the question is here, why we need here a test at all. We pass just children to this component and i think to test this has no value for me.
To test css properties for the overlay is not a good approach.
The only thing we could test is the version info which is only for development.
My suggestion is to remove the test to be honest

Reply:
Fine as well. But I just think whenever a snapshot test is updated instead it should either be removed or replaced, the choice is up to you :)

Reply:
rm test